README.md

```diff
@@ -972,6 +972,12 @@ If you do not want a redirect, but want the error-page rendered on the current-u
 $request->setRewriteCallback('ErrorController@notFound');
 ```
 
+If you will set the correct status for the browser error use:
```

Comment:
What do you think about: "If you want to set a HTTP-Statuscode for a response use:"